packages/core/src/amazonq/errors.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { ToolkitError } from '../shared/errors'
+
+export class CommonAmazonQContentLengthError extends ToolkitError {
+    constructor(message: string) {
+        super(message, { code: 'CommonAmazonQContentLengthError' })
```

Comment:
Having this be the code doesn't convey any extra information
```suggestion
        super(message, { code: 'ContentLengthError' })
```